index.js

```diff
@@ -152,7 +155,7 @@ class LiveReloadPlugin {
     if (
         this._isRunning()
         && include.length > 0
```

Comment:
I just tested this code and in my case `include.length` is already 0 since none of the "assets" in the compilation has changed. In my scenario the shouldReload-call should need to "winn" over the include as well.